jsonrpc/src/main.ts

```diff
@@ -205,6 +206,36 @@ export namespace Trace {
 	}
 }
 
+export enum TraceFormat {
```

Comment:
Instead of having a number enum and a TraceFormatValues would it make sense to use a string Enum which is now supported by TS?

Reply:
Was aligning to `TraceValues`, string enum makes sense.